extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/photos/GooglePhotosExporter.java

```diff
@@ -122,6 +122,9 @@ public GooglePhotosExporter() {
       }
 
       ResultType resultType = ResultType.CONTINUE;
+      if (nextPageData == null || continuationData.getContainerResources().isEmpty()) {
```

Comment:
I have a question here, if there is container should the result type be Continue or end?